### PR TITLE
Populate README with training example

### PR DESCRIPTION
**Pull request by sayakpaul**
Some stuff is copy-pasted from https://github.com/huggingface/transformers/tree/main/examples/research_projects/jax-projects and https://github.com/huggingface/diffusers/tree/main/examples/controlnet.

### CODE REVIEW

File: jax-controlnet-sprint/README.md [hunk 1]
```diff
@@ -177,3 +183,164 @@ You can refer to the notebook to create your own datasets using other MediaPipe
...
+```
+
+To verify that JAX was correctly installed, you can run the following command:
+
+```python
+import jax
+jax.device_count()
+```
+
+This should display the number of TPU cores, which should be 8 on a TPUv4-8 VM.
```
Comment (yiyixuxu) [inline jax-controlnet-sprint/README.md:259]:
```suggestion
This should display the number of TPU cores, which should be 4 on a TPUv4-8 VM.
```

File: jax-controlnet-sprint/README.md [hunk 2]
```diff
@@ -177,3 +183,164 @@ You can refer to the notebook to create your own datasets using other MediaPipe
...
+```
+
+Then clone this repository and install the other dependencies:
+
+```bash
+$ git clone https://github.com/huggingface/community-events
+$ cd community-events/training_scripts
+$ pip install -U -r requirements_flax.txt
+```
+
```
Comment (yiyixuxu) [inline jax-controlnet-sprint/README.md:274]:
```suggestion
If you want to use Weights and Bias logging, you should also install `wandb` now

```bash
pip install wandb
```

```

**Review by yiyixuxu — APPROVED** [review]
awesome - thank you!

I just make a PR to make a small update in the training script (it fixes a bug that's not super important) https://github.com/huggingface/diffusers/pull/2986

we can make a separate PR to update it here after you review it

File: jax-controlnet-sprint/README.md (hunk 1 above)
Comment (sayakpaul) [reply jax-controlnet-sprint/README.md:259]: Shouldn't it be 8? :o

Comment (sayakpaul) [reply jax-controlnet-sprint/README.md:259]: Ah, sorry. Yeah 4 cores.

**Pull request merged**